### ISSUE TITLE
Service Clone

### ISSUE THREAD

**Issue by petermezei** [slamby/slamby-insight#19]
Clone service with given settings.
Workflow to fit in:
1. Select a given service.
2. Click the actions list. Here we should put a clone options. Clone should create a new service, copying the name, and description, with no alias and with a new id. Clone should clone the preparation settings. The new service can be modified (name, description, alias), and the preparation setting should be filled, just like cloning a dataset.

Another suggested way:
1. Create a new service,
2. Instead of prepare, use a prepare by clone, where we can select a source service, and the preparation window pops up copying the source settings.

@attilalaszlo @sbeci  any idea? Please validate it before start.

**Comment by attilalaszlo** [comment]
The first approach can be problematic because you can't modify the prepare settings on the "new service" modal. Actually you can not see the prepare or activate settings on that.
Maybe second way can be easier.

**Comment by petermezei** [comment]
Server: demo server
Services: 3 demo services predefined.

Tested situations:
- Search-clone: works for preparation clone but activation clone **does not work**,
- Classifier-clone: works for preparation and activation clone,
- PRC Service clone: works for preparation clone but activation clone **does not work**

PRC & Search Activation settings clone issue:
----

I opened the service for activation, I select the clone option, select the source service, but the Ok button switches to disabled state and it's not changing.

![image](https://cloud.githubusercontent.com/assets/18461114/23162873/61406ed4-f830-11e6-8080-e306e1352fee.png)

**When I click the clone button, an Internal Error message pops: `Cannot read property 'Filter of undefined'`**

![image](https://cloud.githubusercontent.com/assets/18461114/23162921/9b1e8474-f830-11e6-89c3-4d1f8ba04e6f.png)


@attilalaszlo it might be a bug.
